addons/mail/models/discuss/discuss_channel_member.py

```diff
@@ -318,7 +324,7 @@ def _rtc_join_call(self, store: Store = None, check_rtc_session_ids=None, camera
                     "serverInfo": self._get_rtc_server_info(rtc_session, ice_servers),
                 },
             )
-        if len(self.channel_id.rtc_session_ids) == 1:
+        if len(self.channel_id.rtc_session_ids) == 1 and self._should_invite_members_to_join_call():
```

Comment:
Isn't the method misplaced according to its name?
I'd expect it to be around `self._rtc_invite_members()` and not more.

Reply:
We don't want the notification posted if we don't ring the user?
To be fair, I'm not sure about that, the call started anyway, so the notification might make senses.

Reply:
Let's go with not posting the notification for now, we can always re-add it in this flow if we feel it is problematic to not have it.

Reply:
After testing I changed my mind, we no longer display a notification for this kind of message so it is not problematic to have it.

Moreover the current diff removed it from channels for which it was fine to have.

I'll adapt the condition to simply keep this notification. If after testing we still want to also remove this notification for calendar events, we can make another condition to override.